scout/server/blueprints/public/templates/public/index.html

```diff
@@ -46,8 +44,18 @@ <h1 class="display-4">Scout</h1>
               <button type="submit" class="btn btn-primary form-control text-white">Login</button>
             </div>
           </div>
-        </form>
-      {% endif %}
+        {% endif %}
+        <!-- Consent Banner -->
```

Comment:
Shouldn't we always show a notice saying something like
"Scout uses session cookies. They are necessary for your usage experience."

Reply:
Yeah we can add it, but cookies are a different thing and I think that type of banner should be shown always. Let's have on a different PR, to keep this one more simple.

Reply:
Yes, it should always be there! I was thinking it would just be one line of text that would anyway sit together with this new one. But we could do the EU-cookie banner in a separate banner as well.